<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanetaryImagerMainWindow</class>
 <widget class="QMainWindow" name="PlanetaryImagerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planetary Imager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="image" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>C&amp;amera</string>
    </property>
    <widget class="QMenu" name="menu_device_load">
     <property name="title">
      <string>&amp;Open</string>
     </property>
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/resources/photo.png</normaloff>:/resources/photo.png</iconset>
     </property>
    </widget>
    <addaction name="action_devices_rescan"/>
    <addaction name="menu_device_load"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>V&amp;iew</string>
    </property>
    <addaction name="actionEdges_Detection"/>
    <addaction name="actionStretch_histogram"/>
    <addaction name="actionStretch_colour_saturation"/>
   </widget>
   <widget class="QMenu" name="menuROI">
    <property name="title">
     <string>ROI</string>
    </property>
    <addaction name="actionSelect_ROI"/>
    <addaction name="actionClear_ROI"/>
   </widget>
   <widget class="QMenu" name="menuPlanetary_Imager">
    <property name="title">
     <string>P&amp;lanetary Imager</string>
    </property>
    <addaction name="actionShow_settings"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_log_file_folder"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTracking">
    <property name="title">
     <string>Trac&amp;king</string>
    </property>
    <addaction name="actionAddTrackingTarget"/>
    <addaction name="actionSetCentroidArea"/>
    <addaction name="separator"/>
    <addaction name="actionDisableTracking"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <addaction name="menuPlanetary_Imager"/>
   <addaction name="menuDevice"/>
   <addaction name="menuView"/>
   <addaction name="menuROI"/>
   <addaction name="menuTracking"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="chipInfoWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>&amp;Camera Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="camera_settings">
   <property name="windowTitle">
    <string>Ca&amp;mera Controls</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QScrollArea" name="settings_container">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>80</width>
          <height>133</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="recording">
   <property name="windowTitle">
    <string>&amp;Recording</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_3"/>
   </widget>
  </widget>
  <widget class="QToolBar" name="imageManipulation">
   <property name="windowTitle">
    <string>Main</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEdges_Detection"/>
   <addaction name="actionNight_Mode"/>
   <addaction name="actionShow_settings"/>
  </widget>
  <widget class="QToolBar" name="ROIToolbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>ROI</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect_ROI"/>
   <addaction name="actionEdit_ROI"/>
   <addaction name="actionClear_ROI"/>
  </widget>
  <widget class="QDockWidget" name="histogram">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Histo&amp;gram</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_4"/>
   </widget>
  </widget>
  <widget class="QToolBar" name="trackingToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Tracking</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddTrackingTarget"/>
   <addaction name="actionSetCentroidArea"/>
   <addaction name="actionDisableTracking"/>
  </widget>
  <widget class="QDockWidget" name="mount">
   <property name="windowTitle">
    <string>Mo&amp;unt</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5"/>
  </widget>
  <action name="action_devices_rescan">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/reload.png</normaloff>:/resources/reload.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rescan</string>
   </property>
  </action>
  <action name="action_device_use">
   <property name="text">
    <string>&amp;Load...</string>
   </property>
  </action>
  <action name="actionDevices">
   <property name="text">
    <string>Devices</string>
   </property>
  </action>
  <action name="actionChip_Info">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Chip Info</string>
   </property>
  </action>
  <action name="actionCamera_Settings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera &amp;Settings</string>
   </property>
  </action>
  <action name="actionRecording">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Recording</string>
   </property>
  </action>
  <action name="actionStart_Live">
   <property name="text">
    <string>Start Live</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/delete.png</normaloff>:/resources/delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionPickFolder">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/folder.png</normaloff>:/resources/folder.png</iconset>
   </property>
   <property name="text">
    <string>Pick</string>
   </property>
  </action>
  <action name="actionHide_all">
   <property name="text">
    <string>&amp;Hide all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionShow_all">
   <property name="text">
    <string>Sh&amp;ow all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
  <action name="actionShow_settings">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/gear.png</normaloff>:/resources/gear.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configuration</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Planetary Imager</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionEdges_Detection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/action_edges_detection.png</normaloff>:/resources/action_edges_detection.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edges Detection</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>H&amp;istogram</string>
   </property>
  </action>
  <action name="actionSelect_ROI">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/roi.png</normaloff>:/resources/roi.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Select ROI</string>
   </property>
  </action>
  <action name="actionClear_ROI">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/undo_roi.png</normaloff>:/resources/undo_roi.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear ROI</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Q&amp;uit</string>
   </property>
  </action>
  <action name="actionNight_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/night_mode.png</normaloff>:/resources/night_mode.png</iconset>
   </property>
   <property name="text">
    <string>Night Mode</string>
   </property>
  </action>
  <action name="actionScripting">
   <property name="text">
    <string>&amp;Scripting</string>
   </property>
  </action>
  <action name="actionOpen_log_file_folder">
   <property name="text">
    <string>&amp;Open log file folder...</string>
   </property>
  </action>
  <action name="actionEdit_ROI">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/edit_roi.png</normaloff>:/resources/edit_roi.png</iconset>
   </property>
   <property name="text">
    <string>Edit ROI</string>
   </property>
  </action>
  <action name="actionStretch_histogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Histogram equalization</string>
   </property>
  </action>
  <action name="actionStretch_colour_saturation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Stretch colour saturation</string>
   </property>
  </action>
  <action name="actionAddTrackingTarget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Add tracking target</string>
   </property>
   <property name="toolTip">
    <string>Add tracking target</string>
   </property>
  </action>
  <action name="actionSetCentroidArea">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Set centroid area</string>
   </property>
  </action>
  <action name="actionDisableTracking">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Disable tracking</string>
   </property>
  </action>
  <action name="actionMount">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mount</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
